drivers/sensor/current_amp/current_amp.c

```diff
@@ -53,6 +53,10 @@ static int get(const struct device *dev, enum sensor_channel chan, struct sensor
 		return -ENOTSUP;
 	}
 
+	if (raw_val < config->noise_threshold) {
```

Comment:
Looks like `raw_val` is signed but `noise_threshold` is unsigned, can you clarify? Should we take the abs value of `raw_val`?

Reply:
You are right, I missed this since I am currently only using single-ended ADCs.